internal/config/config.go

```diff
@@ -72,6 +74,7 @@ func Parse(configPath string) (*Config, error) {
 
 	overrideWithEnvVar("HTTP_PROXY", &config.HTTPProxyURL)
 	overrideWithEnvVar("HTTPS_PROXY", &config.HTTPSProxyURL)
+	overrideWithEnvUint32("NOZZLE_CLOUDCONTROLLERAPIBATCHSIZE", &config.CloudControllerAPIBatchSize)
```

Comment:
Can we separate the words with `_`
```suggestion
	overrideWithEnvUint32("NOZZLE_CLOUD_CONTROLLER_API_BATCH_SIZE", &config.CloudControllerAPIBatchSize)
```

Reply:
Fixed (I also had to update the test, which is why I didn't accept this suggestion, I'd still have to make the test change anyway).